saffron/src/utils.rs

```diff
@@ -57,17 +56,6 @@ pub struct QueryBytes {
     pub len: usize,
 }
 
-/// For testing purposes
-impl QueryBytes {
```

Comment:
we don't need this anymore since we can use the combinators from proptest to generate exactly what we want